apps/namadillo/src/App/Common/TransactionReceipt.tsx

```diff
@@ -137,10 +133,6 @@ const TransferTransactionReceipt = ({
 export const TransactionReceipt = ({
   transaction,
 }: TransactionReceiptProps): JSX.Element => {
-  const isTransferTransaction = (): boolean => {
-    return allTransferTypes.includes(transaction.type);
-  };
```

Comment:
This condition is not useful, we only show the receipt (this component) for transfer transactions, and the typescript do the job of checking the types